bigbone/src/main/kotlin/social/bigbone/api/method/OAuthMethods.kt

```diff
@@ -61,7 +59,38 @@ class OAuthMethods(private val client: MastodonClient) {
                 append("client_secret", clientSecret)
                 append("redirect_uri", redirectUri)
                 append("code", code)
-                append("grant_type", grantType)
+                append("grant_type", "authorization_code")
+            }
+        )
+    }
+
+    /**
+     * Obtain an access token using OAuth 2 password grant type. To be used during API calls that are not public.
```

Comment:
I still think that the Kdoc should point out the limits of using this method. The OAuth "spec compliance" page points out that this can be used "for bots and other single-user applications", while the actual API methods page does explicitly mention two other grant_types but not this one. Perhaps at least something like:

> Obtain an access token using OAuth 2 password grant type, to be used for bots and other single-user applications. Where possible, [getAccessTokenWithAuthorizationCodeGrant] should be used instead.

Reply:
Absolutely. Sorry forgot to add. Will add it later today 👍🏻